code/__DEFINES/sound.dm

```diff
@@ -1,2 +1,3 @@
 //max channel is 1024. Only go lower from here, because byond tends to pick the first availiable channel to play sounds on
-#define CHANNEL_LOBBYMUSIC 1024
\ No newline at end of file
+#define CHANNEL_LOBBYMUSIC 1024
+#define CHANNEL_ADMIN 1023

```

Comment:
These are not bitflags. [Ref](http://www.byond.com/docs/ref/info.html#/sound/var/channel)